### DISCUSSION TITLE
GitHub student pack approved but not getting benifits.

### DISCUSSION THREAD

**Discussion by DavidNingthou** [community/community#110606] [GitHub Education] [answered]
### Hi everyone,

It shows that I've been approved for GitHub student developer pack 3 days ago but why i haven't got my benifits yet? How long does it take?

**Comment by samus-aran** [comment, marked as answer]
Hey @DavidNingthou  👋

When you submit your application, you'll have your academic affiliation verified and gain access to Global Campus right away. Your educational benefits will arrive within 72 hours of your approval status. However, during busy periods like the start of a new semester, it may take a bit longer.

You’ll get an email notification once your academic benefits are ready, or you can check your status [here](https://education.github.com/discount_requests/pack_application). Meanwhile, as a verified [GitHub Global Campus](https://education.github.com/globalcampus/student) student, you can explore the platform, connect with a local [Campus Expert](https://education.github.com/students/experts), attend events hosted by student leaders and GitHub Education, discover and contribute to student-created repositories from [GitHub Community Exchange](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/about-github-community-exchange) and much more!

You are still within your 72 hours so if you haven't received your benefits within 5 days of your approval status then we recommend you reply to this discussion 👍

**Reply by samus-aran** [reply]
That's awesome @DavidNingthou  ! 🎉 Thanks so much for coming back and sharing the update (by marking the answer 😉 ). This will help other community members seeking similar answers. Start enjoying more of GitHub Global Campus 🌟:

- Connect with a local Campus Expert. For more information on campus experts, see "[About Campus Experts](https://education.github.com/students/experts)."
- Explore and claim offers for free industry tools from the [Student Developer Pack](https://education.github.com/pack).
- See upcoming in-person and virtual events for students, curated by GitHub Education and student leaders.
- View assignments from [GitHub Classroom](https://classroom.github.com/) with upcoming due dates.
- Stay in the know with [Campus TV](https://www.twitch.tv/githubeducation) episodes. Campus TV is created by GitHub and student community leaders and can be watched live or on demand.
- Discover and contribute to student-created repositories from GitHub Community Exchange. For more information, see "[About GitHub Community Exchange](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/about-github-community-exchange)."

It's great to have you in the GitHub Education community and we hope to see you around 👋

**Comment by DavidNingthou** [comment]
Thank you. I just got it now. Seems like posting here made it faster?

**Reply by samus-aran** [reply]
@DavidNingthou I'm sure folks would love to hear that but no, posting in the community does not speed up the approval time period. 😅  Glad you received your academic benefits 🎉 Enjoy and share what you learn.
